Documentation/subcommands/prepare.md

```diff
@@ -52,12 +52,16 @@ c9fad0e6-8236-4fc2-ad17-55d0a4c7d742
 
 | Flag | Default | Options | Description |
 | --- | --- | --- | --- |
+| `--annotation` | none | annotation add to the app's CRIAnnotations field | Set the app's annotations (example: '--annotation=foo=bar'). |
 | `--caps-remove` | none | capability to remove (example: '--caps-remove=CAP\_SYS\_CHROOT,CAP\_MKNOD') | Capabilities to remove from the process's capabilities bounding set, all others from the default set will be included |
 | `--caps-retain` | none | capability to retain (example: '--caps-remove=CAP\_SYS\_ADMIN,CAP\_NET\_ADMIN') | Capabilities to retain in the process's capabilities bounding set, all others will be removed |
+| `--environment` | none | environment variables add to the app's environment variables | Set the app's environment variables (example: '--environment=foo=bar'). |
 | `--exec` | none | Path to executable | Override the exec command for the preceding image. |
 | `--group` | root | gid, groupname or file path | Group override for the preceding image (example: '--group=group') |
 | `--inherit-env` | `false` | `true` or `false` | Inherit all environment variables not set by apps. |
+| `--label` | none | label add to the apps' CRILabels field | Set the app's labels (example: '--label=foo=bar'). |
```

Comment:
I'd prefer to avoid calling this just "label", as it is a quite generic name that we already use for "image label". As this looks more like a user-controlled field for custom data, I'd call it something like "user-label" or "run-label" or "extra-label".